### DISCUSSION TITLE
Exclusion Manger Options

### DISCUSSION THREAD

**Discussion by emtee40** [d4rken-org/sdmaid-se#342] [Ideas]
I know through AppControl you can add exclusions per app but could an 'Add' option menu be added in the Exclusion Manager in Settings? As redundant of an idea it is, but I was checking if I added an exclusion and figured I'd just add it and realized I had to exit back out and go into the APP menu area and do it there.
But, this idea is also related to me still getting used to navigation of the app.

**Comment by d4rken** [comment]
How would this `Add` option work? Just a text input? A list? A shortcut back to AppControl?
